apps/condo/k6/src/registerBillingReceipt.test.ts

```diff
@@ -0,0 +1,155 @@
+import { faker } from '@faker-js/faker/locale/ru'
+import dayjs from 'dayjs'
+import { check } from 'k6'
+import crypto from 'k6/crypto'
+import encoding from 'k6/encoding'
+
```

Comment:
You need to create 5000 properties before checking out will the mutation works faster or slower

Reply:
Good point. Will try